packages/dds/tree/src/simple-tree/api/tree.ts

```diff
@@ -77,21 +77,8 @@ export interface ViewableTree {
 	 * Only one schematized view may exist for a given ITree at a time.
 	 * If creating a second, the first must be disposed before calling `viewWith` again.
 	 *
-	 *
-	 * TODO: Provide a way to make a generic view schema for any document.
+	 * @privateRemarks
 	 * TODO: Support adapters for handling out-of-schema data.
-	 *
-	 * Doing initialization here allows a small API that is hard to use incorrectly.
```

Comment:
This is a justification of an API decision we pivoted away from when initialize was added as a separate API.